amundsen_application/static/js/components/SearchPage/PopularTables/tests/index.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import {
+  POPULAR_TABLES_INFO_TEXT,
+  POPULAR_TABLES_LABEL,
+  POPULAR_TABLES_SOURCE_NAME,
+} from '../constants';
+import InfoButton from 'components/common/InfoButton';
+import SearchList from 'components/SearchPage/SearchList';
+import { PopularTables, PopularTablesProps } from '../';
+
+describe('PopularTables', () => {
+  const setup = (propOverrides?: Partial<PopularTablesProps>) => {
+    const props: PopularTablesProps = {
+      popularTables: jest.fn() as any,
+      getPopularTables: jest.fn(),
+      ...propOverrides
+    };
+    const wrapper = shallow<PopularTables>(<PopularTables {...props} />)
+
+    return { props, wrapper };
+  };
+  
+  describe('render', () => {
```

Comment:
Missing tests for `componentDidMount`, `mapStateToProps`, and `mapDispatchToProps`.